src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ManageOffencesApiClient.kt

```diff
@@ -59,5 +61,25 @@ class ManageOffencesApiClient(@Qualifier("manageOffencesApiWebClient") private v
       .block() ?: throw CouldNotGetMoOffenceInformation("Sds early release exclusions schedule otherwise failed for offence lookup failed for $offencesList, cannot proceed to perform a sentence calculation")
   }
 
+  fun getOffences(offenceCodes: List<String>): List<Offence> {
+    val offenceCodeString = offenceCodes.joinToString(",")
```

Comment:
We may want to check the number of codes before making the request. Perhaps add a const for the max size, then run an async request for separate requests a splint list.

This would be to prevent the URI string being too long to process.